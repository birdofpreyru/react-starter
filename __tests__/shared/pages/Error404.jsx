import { GlobalStateProvider, JU } from '@dr.pogodin/react-utils';

import Error404Page from 'pages/Error404';

test('Matches shallow snapshot', () => {
  JU.snapshot(
    <GlobalStateProvider>
      <Error404Page />
    </GlobalStateProvider>,
  );
});
