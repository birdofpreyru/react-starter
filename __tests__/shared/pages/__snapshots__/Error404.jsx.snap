// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches shallow snapshot 1`] = `
<div
  className="-dr-pogodin-react-utils___src-shared-components-PageLayout-base-theme___container___T3cuHB"
>
  <div
    className="-dr-pogodin-react-utils___src-shared-components-PageLayout-base-theme___sidePanel___lqNh4h "
  />
  <div
    className="-dr-pogodin-react-utils___src-shared-components-PageLayout-base-theme___mainPanel___pPlQO2"
  >
    <h1>
      Error 404: Requested resource is not found.
    </h1>
  </div>
  <div
    className="-dr-pogodin-react-utils___src-shared-components-PageLayout-base-theme___sidePanel___lqNh4h "
  />
</div>
`;
